libmscore/barline.cpp

```diff
@@ -1037,6 +1097,20 @@ void BarLine::layout()
                         // r |= symBbox(SymId::reversedBracketBottom).translated(0, y2);
                         }
                         break;
+                  case BarLineType::END_START_REPEAT:
```

Comment:
Here we probably do need the code to make sure the bbox gets sets right, otherwise the palette element might not be positioned well.